suite-native/module-device-onboarding/src/components/wallet-backup/CardContent.tsx

```diff
@@ -0,0 +1,73 @@
+import { HStack, Text, VStack } from '@suite-native/atoms';
+import { Icon } from '@suite-native/icons';
+import { Translation } from '@suite-native/intl';
+
+import { WalletBackupType } from '../../../hooks/useWalletBackupPicker';
+
+interface CardContentProps {
```

Comment:
nit:  for consistency
```suggestion
type CardContentProps {
```

https://www.notion.so/satoshilabs/Typescript-4640c944a60442e684b45f343104de37?pvs=4#288da7cd9b7c46b180f5391d7ea48d74

Reply:
https://github.com/trezor/trezor-suite/pull/17981/commits/47c932e7e017d515a6346a59d9ceca1f2d6292cd